### DISCUSSION TITLE
Here's a script in case you ever need to bulk give students admin rights to their repositories

### DISCUSSION THREAD

**Discussion by acbart** [community/community#142408] [GitHub Education]
### Hi everyone,

This may be a little niche, but I wanted to provide this solution in case anyone else needed it. 

Recently, I had a GitHub Classroom assignment that required students to modify the settings of their repository. Unfortunately, I forgot to check the "Grant students admin rights to their repository". Changing this setting later does not retroactively update their admin rights, and dozens of students had already accepted the repository. Rather than do the manual clicking, I wrote the linked script to automatically go through a list of student github names and grant them admin permission. It's a simple script, but I figured this might save someone else the trouble of doing this down the line:

https://gist.github.com/acbart/e172eb805562de68461a54e0b0e9f201

Basically, you just need to get a personal access token (under Developer Settings in your own github profile - I just used fairly generous permissions), a list of students (easily obtained by using the "Download Grades" in the "Download" button of the the Github classroom assignment), and then the org and repo names. Modify the variables at the top of the script, and it should run in your browser's developer console. Shouldn't take too long and will update you on the progress.

Good luck out there folks!

**Comment by g-simmons** [comment]
thank you!
